js/ccf-app/src/global.ts

```diff
@@ -739,3 +739,19 @@ export interface OpenEnclave {
     endorsements?: ArrayBuffer,
   ): EvidenceClaims;
 }
+
+export interface SnpAttestationResult {
+  measurement: string;
```

Comment:
These should be exposed as ArrayBuffers, they're not necessarily legal JS strings, so they'd need to be base64'd or hex'ed, and the application code can do that if it wants.

Reply:
Plus it'd be consistent with what we expose the claims with the OE version already.